packages/react/src/ReactElementValidator.js

```diff
@@ -260,7 +260,9 @@ function validatePropTypes(element) {
 function validateFragmentProps(fragment) {
   currentlyValidatingElement = fragment;
 
-  for (const key of Object.keys(fragment.props)) {
+  const keys = Object.keys(fragment.props);
+  for (let i = 0; i < keys.length; i++) {
+    const key = keys[i];
```

Comment:
Strictly saying this is the only case where it matters. But I figured changing TestRenderer code doesn't hurt too.